src/locales/de/battle.ts

```diff
@@ -55,5 +55,7 @@ export const battle: SimpleTranslationEntries = {
   "skipItemQuestion": "Bist du sicher, dass du kein Item nehmen willst?",
   "notDisabled": "{{pokemonName}}'s {{moveName}} ist\nnicht mehr deaktiviert!",
   "eggHatching": "Oh?",
-  "ivScannerUseQuestion": "IV-Scanner auf {{pokemonName}} benutzen?"
+  "ivScannerUseQuestion": "IV-Scanner auf {{pokemonName}} benutzen?",
+  "drainMessage": " had its\nenergy drained!",
```

Comment:
`"drainMessage": " wurde Energie abgesaugt",`

Reply:
<img width="203" alt="image" src="https://github.com/pagefaultgames/pokerogue/assets/50131232/f4aad107-4b8d-40fb-8f11-bdb714267dd4">

```suggestion
  "drainMessage": " wurde ENergie abgesaugt",
```